__tests__/index.test.js

```diff
@@ -76,3 +77,73 @@ describe("GET /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("PATCH /api/reviews/:review_id", () => {
+  test("should update (increase) the number of votes in the specified review object by the amount given and return the object", () => {
+    return request(app)
+      .patch("/api/reviews/4")
+      .send({ inc_votes: 5 })
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review.votes).toBe(12);
+      });
+  });
+  test("should update (decrease) the number of votes in the specified review object by the amount given and return the object", () => {
+    return request(app)
+      .patch("/api/reviews/4")
+      .send({ inc_votes: -5 })
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review.votes).toBe(2);
+      });
+  });
+  test("returns 400 when passed an object with invalid number", () => {
+    return request(app)
+      .patch("/api/reviews/4")
+      .send({ inc_votes: "five" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid input or ID");
+      });
+  });
+  test("returns 400 when passed an object with other invalid formatting", () => {
+    return request(app)
+      .patch("/api/reviews/4")
+      .send({ votes: 5 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid input format");
+      });
+  });
+  test("returns 400 when votes would update to a negative number", () => {
+    return request(app)
+      .patch("/api/reviews/4")
+      .send({ inc_votes: -15 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe(
+          "No. of votes cannot be negative: current vote count is: 7"
```

Comment:
interesting limitation. Remember you've got this functionality when you get into FE!